packages/shared/src/dto/driver.ts

```
@@ -0,0 +1,31 @@
+  name: string;
+  password: string;
+}
+
+export interface UpdateDriverInfoResponseDTO {
+  message: string;
+  uptime: string;
+}
+
+export interface UpdateDriverInfoErrorResponseDTO {
+  error: string;
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)